docs/api-gateway/08-01-expose-service.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Expose a service
+type: Tutorials
+---
+
+This tutorial shows how to expose service endpoints and configure different allowed HTTP methods for them using the API Gateway Controller.
```

Comment:
```suggestion
This tutorial shows how to expose service endpoints and configure different allowed HTTP methods for them using API Gateway Controller.
```

Reply:
(just because we agreed some time ago to omit definite articles in front of Kyma components)